angularjs-portal-home/src/main/webapp/my-app/layout/partials/welcome.html

```diff
@@ -0,0 +1,8 @@
+<div class="welcome-popup center">
+  <h1>Welcome to the new MyUW. Here are the basics.</h1>
+  <img src="img/myuw-welcome.png">
```

Comment:
The content will not be read by screen reader.
add tabindex="0" id="readWelcomeContent" aria-labelledby="readWelcomeContent" inside the <div> tag will make this work - make the content can be focused by TAB in navigation order and then read out when the focus on it.
And make an unique id name (the id name above is just an example)